config/bolt/contenttypes.yaml

```diff
@@ -204,6 +204,34 @@ blockquotes:
     icon_many: "fa:quote-left"
     icon_one: "fa:quote-left"
     viewless: true
+
+showcases:
+    name: Showcases
+    singular_name: Showcase
+    fields:
+        title:
+            type: text
+            class: large
+        slug:
+            type: slug
+            uses: title
+        website_name:
+            type: text
+        website_link:
+            type: text
+            label: Link
+            placeholder: 'http://example.org/'
+        screenshot:
+            type: image
+        publication_date:
+            type: date
+            mode: datetime
+        contributor:
+            type: text
+        comment:

```

Comment:
Do we use this `comment` field anywhere?

Reply:
No, not at the moment. We have to look at this, because if we use the comment field (redactor) the page is out of line.

Reply:
Nice, let's kill it with fire then 🔥